<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DICOM  Image Editor (DIE)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnOpenImageLeft">
          <property name="text">
           <string>Open image(s)</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/button/resources/buttons/document-open.png</normaloff>:/button/resources/buttons/document-open.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listViewLeft"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnOpenImageRight">
          <property name="text">
           <string>Open image(s)</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/button/resources/buttons/document-open.png</normaloff>:/button/resources/buttons/document-open.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listViewRigth"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCopy">
    <property name="title">
     <string>Copy</string>
    </property>
    <widget class="QMenu" name="menuHeader">
     <property name="title">
      <string>Header</string>
     </property>
     <addaction name="action1_to_1"/>
     <addaction name="action1_to_All"/>
    </widget>
    <addaction name="menuHeader"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCopy"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="toolTip">
            <string>Save header changes</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/button/resources/buttons/document-save.png</normaloff>:/button/resources/buttons/document-save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_4">
           <property name="toolTip">
            <string>Undo</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/button/resources/buttons/edit-undo.png</normaloff>:/button/resources/buttons/edit-undo.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="toolTip">
            <string>Redo</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/button/resources/buttons/edit-redo.png</normaloff>:/button/resources/buttons/edit-redo.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMoveUp"/>
   <addaction name="actionMoveDown"/>
  </widget>
  <action name="action1_to_1">
   <property name="text">
    <string>1-to-1</string>
   </property>
  </action>
  <action name="action1_to_All">
   <property name="text">
    <string>1-to-All</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/button/resources/buttons/arrow-up.png</normaloff>:/button/resources/buttons/arrow-up.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/button/resources/buttons/arrow-down.png</normaloff>:/button/resources/buttons/arrow-down.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/button/resources/buttons/help-about.png</normaloff>:/button/resources/buttons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
